### DISCUSSION TITLE
Add ability to update Title and Description with Meta Data to Publications

### DISCUSSION THREAD

**Discussion by RyanCrew121** [datarhei/restreamer#573] [Ideas]
Would like the ability to update one or all publications title and descriptions from the UI panel.  This would unify the stream across all platforms and prevent the user from manually setting live data on each platform.

**Comment by svenerbeck** [comment]
Hello @RyanCrew121 
Thank you for your input. We added this to the FR list: https://github.com/datarhei/restreamer/discussions/345

Cheerio
Sven

**Comment by barroudjo** [comment]
This is a great idea !
